go.sum

```diff
@@ -501,6 +501,8 @@ github.com/opencontainers/go-digest v1.0.0-rc1/go.mod h1:cMLVZDEM3+U2I4VmLI6N8jQ
 github.com/opencontainers/image-spec v1.0.1/go.mod h1:BtxoFyWECRxE4U/7sNtV5W15zMzWCbyJoFRP3s7yZA0=
 github.com/openshift/api v0.0.0-20210831091943-07e756545ac1 h1:BleifEWC+NP/YhYHyQlGrDflXZPxawwOzyLUI+nr4jw=
 github.com/openshift/api v0.0.0-20210831091943-07e756545ac1/go.mod h1:RsQCVJu4qhUawxxDP7pGlwU3IA4F01wYm3qKEu29Su8=
+github.com/openshift/api v0.0.0-20210906075240-3611f00b94fd h1:lHl8nlAfRKsucYBdGK4gJdK/CvaNSGheAsbxhOBm+dU=
+github.com/openshift/api v0.0.0-20210906075240-3611f00b94fd/go.mod h1:RsQCVJu4qhUawxxDP7pGlwU3IA4F01wYm3qKEu29Su8=
```

Comment:
Did you run go mod tidy?

Reply:
I did not, is there anything else I should run before I submit that change?

Reply:
I do `go mod tidy` and `go mod vendor`.

Do you mind explaining the reason behind the change as well? Thanks!